tools/nl/titles/gen_chart_titles_by_sv.py

```diff
@@ -5,27 +5,39 @@
 import pandas as pd
 
 SHEET_URL = 'https://docs.google.com/spreadsheets/d/1lmNAnqECpcvkuOlIkdo50Ve1KAalOoyP_lUlOuLmIAU'
-SHEET_NAME = 'Main'
-CSVFILE = 'demo_svs_1.3k_names.csv'
-JSONFILE = '../../../server/config/nl_page/chart_titles_by_sv.json'
+SHEET_LIST = [{
```

Comment:
Update README.md in this directory to mention there is an additional SDG-specific sheet?

Reply:
done